docs/source/schema/unions-interfaces.md

```diff
@@ -166,3 +189,33 @@ query GetBooks {
   }
 }
 ```
+
+This query uses [inline fragments](https://graphql.org/learn/queries/#inline-fragments) to fetch a `Book`'s `courses` (if it's a `Textbook`) or its `colors` (if it's a `ColoringBook`).
+
+For more information, see [Using fragments with unions and interfaces](https://www.apollographql.com/docs/react/data/fragments/#using-fragments-with-unions-and-interfaces).
+
+
+### Resolving an interface field
+
+[As with union types](#resolving-a-union-field), Apollo Server requires interfaces to define a `__resolveType` function to determine which valid object type an interface field contains.
```

Comment:
Suggesting this based on a previous suggestion I made, but feel free to ignore both of them if you think it adds value.

```suggestion
[As with union types](#resolving-a-union-field), Apollo Server requires interfaces to define a `__resolveType` function to determine which object type an interface field contains.
```